### PR TITLE
264 update seviri reader to use pypublicdecompwt

### PR DESCRIPTION
**Pull request by jsolbrig**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#264

# Testing Instructions
- Unset `XRIT_DECOMPRESS_PATH`
- Check out this branch.
- Reinstall GeoIPS via `pip install -e .`
- Attempt to read SEVIRI data. Maybe run a test script if available.

# Summary
Enhancements
============

Replace xRITDecompress with pyPublicDecompWT for seviri_hrit reader
-------------------------------------------------------------------

*From NRLMMD-GEOIPS/geoips#264: 2023-08-16, Update seviri reader to use pyPublicDecompWT*

* We had previously been using xRITDecompress which needed to be complied and installed
  separately. This replaces xRITDecompress with pyPublicDecompWT which provides the same
  functionality but can be pip installed.

::

    modified: geoips/plugins/modules/readers/utils/hrit_reader.py

Installation Updates
====================

Add pyPublicDecompWT to dependencies and remove setup_seviri from setup script
------------------------------------------------------------------------------

*From NRLMMD-GEOIPS/geoips#264: 2023-08-16, Update seviri reader to use pyPublicDecompWT*

* Add pypublicdecompwt to install requirements
* Remove setup_seviri from setup.py
* Remove xRITDecompress environment variables from config_geoips

::

    modified: pyproject.toml
    modified: setup_seviri
    modified: config/config_geoips

# Output

### CODE REVIEW

**Review by mindyls — APPROVED** [review]
It worked!

I merged my last branch, add_user_modify_config, into this branch and resolved conflicts (and updated release notes) in order to test.  Make sure PRs are merged into main in order.

```
19_075605 single_source1663   INFO: READER_NAME: seviri_hrit
19_075605 single_source1664   INFO: PRODUCT_NAME: WV-Upper
19_075605 single_source1665   INFO: NUM_PRODUCTS: 1
19_075605 single_source1666   INFO: NUM_DELETED_PRODUCTS: 0
19_075605 geoips_utils 311   INFO: Total Time 0:00:29.596680: 1 Num jobs: single_source procflow
19_075605 geoips_utils 321   INFO:     SUCCESS Process Time single_source procflow: self_register       : 0:00:10.688700
19_075605 run_procflow  63   INFO: Completed geoips PROCFLOW single_source processing, done!
19_075605 run_procflow  67   INFO: Starting time: 2023-08-19 14:55:32.211874
19_075605 run_procflow  68   INFO: Ending time: 2023-08-19 14:56:05.870725
19_075605 run_procflow  69   INFO: Total time: 0:00:33.658898
19_075605 run_procflow  89   INFO: Return value: 0
```

NOTE: I had 2 failures!  DOn't have time to look into yet, but could be while I was re-installing geoips...
```
Sat Aug 19 07:38:21 PDT 2023  Running amsr2.config_based_overlay_output.sh
$HOME/geoips/geoips/tests/integration_tests/../..//tests/scripts/amsr2.config_based_overlay_output.sh
$HOME/geoips/outdirs/logs/20230819/20230819.142934_geoips_full/test_all_geoips_full.log_amsr2.config_based_overlay_output.sh.log
        Return: 2

Sat Aug 19 07:39:43 PDT 2023  Running amsr2.config_based_overlay_output_low_memory.sh
$HOME/geoips/geoips/tests/integration_tests/../..//tests/scripts/amsr2.config_based_overlay_output_low_memory.sh
$HOME/geoips/outdirs/logs/20230819/20230819.142934_geoips_full/test_all_geoips_full.log_amsr2.config_based_overlay_output_low_mem
ory.sh.log
        Return: 4
```

Also note the code checks and doc build passed (I had to make one fix, remove blank line after docstring in seviri reader)
```
Sat Aug 19 08:24:08 PDT 2023  Running check_code.sh
./tests/integration_tests/../..//tests/utils/check_code.sh all ./tests/integration_tests/../../ flake8_docstring_only
$HOME/geoips/outdirs/logs/20230819/20230819.152408_geoips_full/test_all_geoips_full.log_check_code.sh__all_._tests_integration_tests_.._..__flake8_docstring_only.log
        Return: 0

Sat Aug 19 08:24:42 PDT 2023  Running build_docs.sh
./tests/integration_tests/../..//docs/build_docs.sh ./tests/integration_tests/../../ geoips html_only
$HOME/geoips/outdirs/logs/20230819/20230819.152408_geoips_full/test_all_geoips_full.log_build_docs.sh__._tests_integration_tests_.._..__geoips_html_only.log
        Return: 0
```

File: docs/source/releases/v1_11_3a0.rst
Comment (mindyls) [inline docs/source/releases/v1_11_3a0.rst:19]: NOTE: Do not merge until all other PRs are merged, and this one is automatically updated to merge to main.

Comment (jsolbrig) [reply docs/source/releases/v1_11_3a0.rst:19]: Done and ready to go.

**Pull request merged**
